### ISSUE TITLE
Different Test Results

### ISSUE THREAD

**Issue by createnameisdiffcult** [yinjunbo/IS-Fusion#32]
Hi, i used your pre-trained model for testing, but obtained different mAP and NDS. Here is  the specific results.

[                                                  ] 1/6019, 0.8 task/s, elapsed: 1s, ETA:  7582s/home/detection24/code/IS-Fusion/mmdet3d/core/bbox/coders/transfusion_bbox_coder.py:99: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  self.post_center_range, device=heatmap.device)
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 6019/6019, 4.9 task/s, elapsed: 1221s, ETA:     0s
Formating bboxes of pts_bbox
Start to convert detection format...
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 6019/6019, 76.3 task/s, elapsed: 79s, ETA:     0s
Results writes to /tmp/tmpyef1krfj/results/pts_bbox/results_nusc.json
Evaluating bboxes of pts_bbox
mAP: 0.7250                                                                                                                                           
mATE: 0.2656
mASE: 0.2516
mAOE: 0.2929
mAVE: 0.2646
mAAE: 0.1850
NDS: 0.7365
Eval time: 46.2s

Per-class results:
Object Class    AP      ATE     ASE     AOE     AVE     AAE
car     0.894   0.162   0.150   0.053   0.266   0.194
truck   0.672   0.307   0.179   0.068   0.245   0.234
bus     0.787   0.312   0.181   0.066   0.456   0.250
trailer 0.518   0.481   0.201   0.454   0.193   0.141
construction_vehicle    0.368   0.660   0.414   1.008   0.144   0.288
pedestrian      0.908   0.125   0.287   0.340   0.213   0.104
motorcycle      0.806   0.180   0.237   0.214   0.385   0.255
bicycle 0.697   0.147   0.256   0.381   0.216   0.015
traffic_cone    0.846   0.099   0.325   nan     nan     nan
barrier 0.755   0.182   0.285   0.052   nan     nan

{'pts_bbox_NuScenes/car_AP_dist_0.5': 0.8162, 'pts_bbox_NuScenes/car_AP_dist_1.0': 0.9012, 'pts_bbox_NuScenes/car_AP_dist_2.0': 0.9236, 'pts_bbox_NuScenes/car_AP_dist_4.0': 0.9333, 'pts_bbox_NuScenes/car_trans_err': 0.1619, 'pts_bbox_NuScenes/car_scale_err': 0.1499, 'pts_bbox_NuScenes/car_orient_err': 0.053, 'pts_bbox_NuScenes/car_vel_err': 0.266, 'pts_bbox_NuScenes/car_attr_err': 0.1939, 'pts_bbox_NuScenes/mATE': 0.2656, 'pts_bbox_NuScenes/mASE': 0.2516, 'pts_bbox_NuScenes/mAOE': 0.2929, 'pts_bbox_NuScenes/mAVE': 0.2646, 'pts_bbox_NuScenes/mAAE': 0.185, 'pts_bbox_NuScenes/truck_AP_dist_0.5': 0.462, 'pts_bbox_NuScenes/truck_AP_dist_1.0': 0.6686, 'pts_bbox_NuScenes/truck_AP_dist_2.0': 0.7599, 'pts_bbox_NuScenes/truck_AP_dist_4.0': 0.7966, 'pts_bbox_NuScenes/truck_trans_err': 0.3067, 'pts_bbox_NuScenes/truck_scale_err': 0.1793, 'pts_bbox_NuScenes/truck_orient_err': 0.0679, 'pts_bbox_NuScenes/truck_vel_err': 0.2448, 'pts_bbox_NuScenes/truck_attr_err': 0.2336, 'pts_bbox_NuScenes/construction_vehicle_AP_dist_0.5': 0.0758, 'pts_bbox_NuScenes/construction_vehicle_AP_dist_1.0': 0.268, 'pts_bbox_NuScenes/construction_vehicle_AP_dist_2.0': 0.4934, 'pts_bbox_NuScenes/construction_vehicle_AP_dist_4.0': 0.6343, 'pts_bbox_NuScenes/construction_vehicle_trans_err': 0.6605, 'pts_bbox_NuScenes/construction_vehicle_scale_err': 0.4137, 'pts_bbox_NuScenes/construction_vehicle_orient_err': 1.0083, 'pts_bbox_NuScenes/construction_vehicle_vel_err': 0.1438, 'pts_bbox_NuScenes/construction_vehicle_attr_err': 0.2878, 'pts_bbox_NuScenes/bus_AP_dist_0.5': 0.5351, 'pts_bbox_NuScenes/bus_AP_dist_1.0': 0.7769, 'pts_bbox_NuScenes/bus_AP_dist_2.0': 0.9047, 'pts_bbox_NuScenes/bus_AP_dist_4.0': 0.9309, 'pts_bbox_NuScenes/bus_trans_err': 0.3125, 'pts_bbox_NuScenes/bus_scale_err': 0.1815, 'pts_bbox_NuScenes/bus_orient_err': 0.0656, 'pts_bbox_NuScenes/bus_vel_err': 0.4564, 'pts_bbox_NuScenes/bus_attr_err': 0.2499, 'pts_bbox_NuScenes/trailer_AP_dist_0.5': 0.197, 'pts_bbox_NuScenes/trailer_AP_dist_1.0': 0.4937, 'pts_bbox_NuScenes/trailer_AP_dist_2.0': 0.6584, 'pts_bbox_NuScenes/trailer_AP_dist_4.0': 0.7213, 'pts_bbox_NuScenes/trailer_trans_err': 0.4813, 'pts_bbox_NuScenes/trailer_scale_err': 0.201, 'pts_bbox_NuScenes/trailer_orient_err': 0.4541, 'pts_bbox_NuScenes/trailer_vel_err': 0.1925, 'pts_bbox_NuScenes/trailer_attr_err': 0.141, 'pts_bbox_NuScenes/barrier_AP_dist_0.5': 0.6661, 'pts_bbox_NuScenes/barrier_AP_dist_1.0': 0.7525, 'pts_bbox_NuScenes/barrier_AP_dist_2.0': 0.7978, 'pts_bbox_NuScenes/barrier_AP_dist_4.0': 0.8046, 'pts_bbox_NuScenes/barrier_trans_err': 0.1821, 'pts_bbox_NuScenes/barrier_scale_err': 0.2849, 'pts_bbox_NuScenes/barrier_orient_err': 0.0519, 'pts_bbox_NuScenes/barrier_vel_err': nan, 'pts_bbox_NuScenes/barrier_attr_err': nan, 'pts_bbox_NuScenes/motorcycle_AP_dist_0.5': 0.6979, 'pts_bbox_NuScenes/motorcycle_AP_dist_1.0': 0.8277, 'pts_bbox_NuScenes/motorcycle_AP_dist_2.0': 0.8453, 'pts_bbox_NuScenes/motorcycle_AP_dist_4.0': 0.8541, 'pts_bbox_NuScenes/motorcycle_trans_err': 0.1804, 'pts_bbox_NuScenes/motorcycle_scale_err': 0.2368, 'pts_bbox_NuScenes/motorcycle_orient_err': 0.2138, 'pts_bbox_NuScenes/motorcycle_vel_err': 0.3845, 'pts_bbox_NuScenes/motorcycle_attr_err': 0.2549, 'pts_bbox_NuScenes/bicycle_AP_dist_0.5': 0.6631, 'pts_bbox_NuScenes/bicycle_AP_dist_1.0': 0.7008, 'pts_bbox_NuScenes/bicycle_AP_dist_2.0': 0.7077, 'pts_bbox_NuScenes/bicycle_AP_dist_4.0': 0.7153, 'pts_bbox_NuScenes/bicycle_trans_err': 0.1471, 'pts_bbox_NuScenes/bicycle_scale_err': 0.2562, 'pts_bbox_NuScenes/bicycle_orient_err': 0.3815, 'pts_bbox_NuScenes/bicycle_vel_err': 0.2164, 'pts_bbox_NuScenes/bicycle_attr_err': 0.015, 'pts_bbox_NuScenes/pedestrian_AP_dist_0.5': 0.8932, 'pts_bbox_NuScenes/pedestrian_AP_dist_1.0': 0.905, 'pts_bbox_NuScenes/pedestrian_AP_dist_2.0': 0.913, 'pts_bbox_NuScenes/pedestrian_AP_dist_4.0': 0.9198, 'pts_bbox_NuScenes/pedestrian_trans_err': 0.1246, 'pts_bbox_NuScenes/pedestrian_scale_err': 0.2871, 'pts_bbox_NuScenes/pedestrian_orient_err': 0.34, 'pts_bbox_NuScenes/pedestrian_vel_err': 0.2128, 'pts_bbox_NuScenes/pedestrian_attr_err': 0.1039, 'pts_bbox_NuScenes/traffic_cone_AP_dist_0.5': 0.8313, 'pts_bbox_NuScenes/traffic_cone_AP_dist_1.0': 0.8386, 'pts_bbox_NuScenes/traffic_cone_AP_dist_2.0': 0.85, 'pts_bbox_NuScenes/traffic_cone_AP_dist_4.0': 0.8655, 'pts_bbox_NuScenes/traffic_cone_trans_err': 0.0992, 'pts_bbox_NuScenes/traffic_cone_scale_err': 0.3252, 'pts_bbox_NuScenes/traffic_cone_orient_err': nan, 'pts_bbox_NuScenes/traffic_cone_vel_err': nan, 'pts_bbox_NuScenes/traffic_cone_attr_err': nan, 'pts_bbox_NuScenes/NDS': 0.7365322181323686, 'pts_bbox_NuScenes/mAP': 0.7250091604474161}
